pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -43,11 +45,26 @@ const (
 
 var runInTestMode bool
 var runInDemoMode bool
+var maxConcurrentReconciles int
 
 func init() {
 	flag.BoolVar(&runInTestMode, "test-mode", false, "disable ironic communication")
 	flag.BoolVar(&runInDemoMode, "demo-mode", false,
 		"use the demo provisioner to set host states")
+
+	if mcrEnv, ok := os.LookupEnv("OPERATOR_CONCURRENCY"); ok {
+		mcr, err := strconv.Atoi(mcrEnv)
+		if err == nil && mcr > 0 {
+			log.Info("OPERATOR_CONCURRENCY is set via an environment variable", mcr)
+			maxConcurrentReconciles = mcr
+		} else {
+			log.Error(err, "OPERATOR_CONCURRENCY value is invalid")
+			os.Exit(1)
+		}
+	} else {
+		log.Info("Operator Concurrency will be set to a default value of 3")
```

Comment:
Could use Sprintf to insert the default value here so it won't have to be changed in 2 places if we ever change the default. (I would probably just set the default on line 48 so nobody ever has to go looking for it.

Reply:
+1. Will apply the change.